float32 x_distance
float32 y_distance
float32 z_distance
int32 ids
float32[4] quaternion
string aruco_type